Sources/MapboxMaps/Style/AttributionDialogManager.swift

```diff
@@ -131,11 +132,18 @@ extension AttributionDialogManager: InfoButtonOrnamentDelegate {
         let bundle = Bundle.mapboxMaps
 
         if let attributions = dataSource?.attributions() {
-            for attribution in attributions {
-                let action = UIAlertAction(title: attribution.title, style: .default) { _ in
-                    self.delegate?.attributionDialogManager(self, didTriggerActionFor: attribution)
+
+            // Non actionable single item get's displayed as alert's title
+            if attributions.count == 1, let attribution = attributions.first, attribution.url == nil {
+                alert.message = attribution.title
+            } else {
+                for attribution in attributions {
+                    let action = UIAlertAction(title: attribution.title, style: .default) { _ in
+                        self.delegate?.attributionDialogManager(self, didTriggerActionFor: attribution)
+                    }
```

Comment:
I actually wonder if we wouldn't be better off changing the signature of the delegate method to take a non-optional URL instead of an Attribution and moving the whole guard statement I linked into here.

Reply:
Yeah, with optional url it looks quite ugly, I introduced `Attribution.Kind` to help with this.